src/graphics/graphics.js

```diff
@@ -666,7 +666,25 @@
         UNIFORMTYPE_TEXTURECUBE: 16,
         UNIFORMTYPE_FLOATARRAY: 17,
 
-        SHADERTAG_MATERIAL: 1
+        SHADERTAG_MATERIAL: 1,
+
+        FUNC_NEVER: 0,
```

Comment:
I ordered the 'enums' in this file alphabetically, at least based on the word before the first underscore. Not a big deal but....